cartridge-cli.lua

```diff
@@ -2351,7 +2386,23 @@ local function check_if_deprecated_build_flow_is_ised(source_dir)
 end
 
 local function app_pack(args)
-    local name, version, release = detect_name_release_version(args.path, args.name, args.version)
+    local name, version, release = detect_name_version_release(args.path, args.name, args.version)
+
+    -- collect general application info
+    pack_state.path = args.path
+    pack_state.name = name
+    pack_state.version = version
+    pack_state.release = release
+    pack_state.version_release = string.format('%s-%s', version, release)
+
+    -- collect pack-specific application info
+    pack_state.dest_dir = fio.abspath('.')
+    pack_state.from = args.from
+    pack_state.download_token = args.download_token
+    pack_state.docker_build_args = args.docker_build_args
+    pack_state.deprecated_flow = check_if_deprecated_build_flow_is_ised()
+    pack_state.tarantool_is_enterprise = tarantool_is_enterprise()
+
```

Comment:
Let's add asserts here to be sure that no one of these parameters is empty. We will exploded before could start some destructive actions